src/pages/video/video.js

```diff
@@ -39,7 +40,7 @@ export default function VideoPage() {
         <StyledMain>
           <VideoCard>
             <div className="card-media">
-              <img src="https://picsum.photos/300/400/" alt="video" />
+              <ReactPlayer url={`https://www.youtube.com/watch?v=${id}`} />
             </div>
           </VideoCard>
           <Details display="flex" gap="sm">
```

Comment:
Better to add watchlater and add to playlist functionality in single video page

Reply:
Ok, will check this